src/main/resources/templates/elasticsearch/caseMatching/dormant_date_range_query.json

```diff
@@ -0,0 +1,51 @@
+{
+  "size": :size,
+  "from": :from,
+  "query": {
+    "bool": {
+      "must_not": { "match": { "state": "Dormant" }},
+      "filter": [
+        {
+          "range": {
+            "last_modified": {
+              "lte": ":fromDate"
+            }
+          }
+        },
+        {
+          "bool": {
+            "should": [
+              { "match": { "state": "BOCaseMatchingExamining" }},
```

Comment:
maybe we should drive this from code? push in a list from java? just a suggestion

Reply:
Added comment in the code